worldedit-core/src/main/java/com/sk89q/worldedit/command/util/AsyncCommandBuilder.java

```diff
@@ -94,6 +96,12 @@ private AsyncCommandBuilder(Callable<T> callable, Actor sender) {
         return this;
     }
 
+    public AsyncCommandBuilder<T> sendDelayedRepeatingMessage(Component message) {
```

Comment:
Shouldn't this be more like `sendMessageAfterFurtherDelay` (maybe `Delays`)? Currently this is in the present tense.